src/main/java/edu/uc/inventorymanager/ui/MainController.java

```diff
@@ -74,17 +76,20 @@ public ResponseEntity fetchItemById(@PathVariable("id") int id) {
     /**
      * Fetch user with given ID
      * <p>
-     * Given the parameter id, find an user that corresponds to this unique id.
+     * Given the parameter id, find a user that corresponds to this unique id.
      * <p>
      * Returns one of the following status codes:
      * 200: user found
-     * 400: user not found
+     * 404: user not found
```

Comment:
Same as with fetchItemById, a more specific error code can be more helpful to the user